amarachukwu-agbo/random-number-generator
# #165145137 Set up Node and Express

#### What does this PR do?
Sets up Node and Express

#### Description of Task to be completed?
- Install node, express and babel packages
- Add commands to start, build and serve the server
- Add babel config file

#### How should this be manually tested?
- Clone the repo and install application dependencies. 
- Run command  `yarn server:start`. You should see the server running on port 3000
- Run command  `yarn server:build`. You should see a new folder in the root directory `dist` which contains the transpiled code

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#165145137](https://www.pivotaltracker.com/story/show/165145137)

#### Screenshots (if appropriate)
N/A

#### Questions: 
N/A
